Expand the `Skin` stub (#9063)
ppy/osu-wiki: wiki/Skin/en.md

# Skin

> A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players use skins to allow for a more customised user experience. Most of the important UI elements and sounds in osu! can be "skinned". 

Reviewer: ```suggestion
A **skin** is a set of images and audio clips that replace the default ones used by osu!. Many players use skins to allow for a more customised user experience. Most of the important UI elements and sounds in osu! can be overridden, or *skinned*.
```

> ## Reasons to Use Skins

Reviewer: ```suggestion
## Reasons to use skins
```

> - Skins can be used to further optimize a player's gameplay, since many skins hide certain visual details or offer gameplay elements focused on a certain playstyle like specific mod combinations.

Reviewer: ```suggestion
- Skins can be used to further optimize gameplay, since many of them hide certain visual details or offer gameplay elements focused on a certain [playstyle](/wiki/Gameplay/Play_style), such as specific [mod](/wiki/Gameplay/Game_modifier) combinations.
```

- In addition to gameplay focused changes, skins can also offer custom menus which commonly feature characters from various anime and games, as well as being based around certain design styles, like minimalism or techno.

## How to Use Custom Skins

> Skins can either be [created from scratch](/wiki/Skinning), or downloaded as a `.osk` file and imported into the game. To import an `.osk` file simply open it with osu!, or drag-and-drop it into osu!. If a skin you download is in `.zip` or `.rar` format unzip it and move it into the `Skins` folder of your osu! installation. Make sure that you are moving the folder containing the `skin.ini` and not a folder that contains the folder.

Reviewer: I think this reads better, besides, it gets rid of all of the `you`s
```suggestion
Skins can either be [created from scratch](/wiki/Skinning), or downloaded as an [`.osk` file](/wiki/Client/File_formats/Osk_(file_format)) and imported into the game. To import an `.osk` file, simply open it with osu!, or drag and drop it into the game. If the downloaded skin file has the `.zip` or `.rar` format, extract. The resulting folder, which contains the `skin.ini` file, can then be moved into the `Skins` folder of your osu! installation.
```

> ## Popular Skin Websites

Reviewer: I don't feel "websites" is the right choice here, provided 3 of 5 links point to the forum threads
```suggestion
## Popular skin collections
```

- [Work In Progress Skins forum](https://osu.ppy.sh/community/forums/119)
- [Remixed Skins forum](https://osu.ppy.sh/community/forums/124)
- [Completed Skins forum](https://osu.ppy.sh/community/forums/109)

> - [Completed Skins Compendium](https://compendium.skinship.xyz/)<!-- TODO: explain how it expands upon completed -->

Reviewer: could you explain the TODO better?

Author: probably worth explaining that is related to the completed skins forum, just didnt know how to word it

- [Circle People Pro Player Skin Collection](https://circle-people.com/skins/)

> Side Note: Most osu! players these days use custom skins, with many of the high-ranking ones either linking them on their userpage, in YouTube video descriptions or providing it via a Twitch chat command.

Reviewer: not sure this is worth mentioning at all

Author: you would be surprised how often a "how do i get this skin <youtube link>" can be solved with "check the description/user profile"

Reviewer: then,
```suggestion
In addition to the above, many players, including the high-ranking ones, post own personal skins on their [userpages](/wiki/osu!supporter#editable-profile-section), in YouTube video descriptions, or provide them via a Twitch chat command.
```

> - See also: [Skinning](/wiki/Skinning)

Reviewer: I think this is irrelevant becuase there's already a link to the skinning article (alternatively, you could move this back to a note under the `# Skin` heading)
```suggestion
```
